### DISCUSSION TITLE
Failed to run esbuild-example

### DISCUSSION THREAD

**Discussion by gangtao** [perspective-dev/perspective#2503] [Q&A] [answered]
Hi,

I really love what your guys are building here.

I want to try with a simple sample but stucked with some error.

when I run the esbuild-example, I got following errors:

```
GET http://localhost:8081/perspective_viewer_bg.wasm 404 (Not Found)
localhost/:13 
GET http://localhost:8081/editor.worker.js 404 (Not Found)
index.js:28912 Uncaught (in promise) TypeError: Failed to execute 'compile' on 'WebAssembly': Incorrect response MIME type. Expected 'application/wasm'.
await (async)
Je @ index.js:30287
await in Je (async)
(anonymous) @ index.js:30289
12index.js:28912 Uncaught (in promise) TypeError: Failed to execute 'compile' on 'WebAssembly': Incorrect response MIME type. Expected 'application/wasm'.
```

my os is Mac OS 14.2.1 (23C71)
my browser is chrome Version 120.0.6099.129 (Official Build) (arm64)

Thanks for helping

**Comment by texodus** [comment, marked as answer]
This is a [bug in the example's `http-server` dependency](https://github.com/http-party/http-server/issues/690), the file server which serves this example uses the wrong `Content-Type` header. We will update this example's dependency version, in the meantime modifying this example to use the newest `http-server` is a workaround (`14.1.1` works for me).

I'm not sure how this regression occurred as this example certainly worked when written, I suppose Chrome tightened what mime types they allow for streaming instantiation in newer versions

**Reply by gangtao** [reply]
Thanks @texodus I will try it

**Reply by gangtao** [reply]
Yes, 14.1.1 works
